controllers/isValidNumLoans.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+var LARService = require('../services/LARService');
+
+module.exports = function(router) {
+
+    /**
+     * @param {String} activityYear, {String} numLoans, {String} respondentID
+     * @return {json}
+     */
+    router.get('/:activityYear/:numLoans/:respondentID', function(req, res) {
```

Comment:
The params respondentID and numLoans should probably be swapped here in the RESTful sense.